app/views/api/services/usage_rules.html.slim

```diff
@@ -0,0 +1,22 @@
+- # TODO: When APIAP is default, adjust the old settings cukes to usage rules
+- content_for :sublayout_title, 'Usage Rules'
+
+- if (can? :manage, :service_plans) && !current_account.settings.service_plans_ui_visible?
+	= form_tag polymorphic_path([:masterize, :admin, @service, ServicePlan]), :method => :post, :class => 'service autosubmit formtastic', :remote => true do
+		fieldset.inputs
+			legend Default Service Plan
+			ol
+				li.string.optional
```

Comment:
Are those tabs?
The indentation is really weird, it should be 2 spaces

Reply:
That's true, I've copy/pasta from the other view... I'll take a look and see if the IDE did some nasty thing..

Reply:
hahahah exactly, i was really doubting if it was just me seeing it weird for using a different laptop and setup now :joy:

Reply:
and yeah, it was the IDE settings. fixed